### DISCUSSION TITLE
New Sequins "Parallel" chaining mode. (All four sequences in a channel play in parallel, rather than sequentially. )

### DISCUSSION THREAD

**Discussion by TheDichotomist** [djphazer/O_C-Phazerville#65] [Ideas]
It's perhaps a strange request. But It would be very interesting if instead of sequentially accessing each channel's 4 sequences, one after the other, or relying on CV address to pick steps out of a basket, if we could have all four playing simultaneously, and switch between them while they're playing and phasing in and out with each other, depending on their lengths or clock settings.

AUX CV could address the sequences, and an AUX trigger could reset them all at once. 

You can see this trick utilized in this video by Omri Cohen. https://youtu.be/Qw5jzmH9bm8?t=430 

If this is already possible, do let me know. 
Thanks for your time!

**Comment by nbeirne** [comment]
I think you can do this with my SwitchSeq app, along with Seq32 to program the sequences and their lengths.

Check out this PR for the discussion: https://github.com/djphazer/O_C-Phazerville/pull/67
